server/tserver/src/test/java/org/apache/accumulo/tserver/log/SortedLogRecoveryTest.java

```diff
@@ -79,6 +80,7 @@
   static final Text cq = new Text("cq");
   static final Value value = new Value("value");
   static ServerContext context;
+  static CryptoService cs = CryptoServiceFactory.none();
```

Comment:
It does not appear this is used anywhere. Can this be removed or am I overlooking something?

Reply:
Yup good catch. This was leftover from a previous change.